### PR TITLE
Clarify Batch and Backup/Restore docs to emphasize immutability #1086

### PR DESCRIPTION
**Pull request by ryanemerson**

### CODE REVIEW

File: documentation/asciidoc/stories/assembly_backing_up_restoring.adoc [hunk 1]
```diff
@@ -8,7 +8,13 @@ ifdef::context[:parent-context: {context}]
 
 `Backup` CR:: Archives {brandname} cluster content to a persistent volume.
 `Restore` CR:: Restores archived content to a {brandname} cluster.
-
++
+[IMPORTANT]
+====
+The `Backup` and `Restore` CRs are one-time events that do not reconcile updates to the `.spec` fields.
+If the `.spec` needs updating or the execution of the backup/restore operation fails, then it's necessary for a new `Batch` CR instance to be created.
```
Comment (dmvolod) [inline documentation/asciidoc/stories/assembly_backing_up_restoring.adoc:15]: Backup and Restore

File: documentation/asciidoc/stories/assembly_backing_up_restoring.adoc [hunk 2]
```diff
@@ -8,7 +8,13 @@ ifdef::context[:parent-context: {context}]
 
 `Backup` CR:: Archives {brandname} cluster content to a persistent volume.
 `Restore` CR:: Restores archived content to a {brandname} cluster.
-
++
+[IMPORTANT]
+====
+The `Backup` and `Restore` CRs are one-time events that do not reconcile updates to the `.spec` fields.
+If the `.spec` needs updating or the execution of the backup/restore operation fails, then it's necessary for a new `Backup` or 'Restore' CR instance to be created.
```
Comment (oraNod) [inline documentation/asciidoc/stories/assembly_backing_up_restoring.adoc:15]:
```suggestion
Applying a `Backup` or `Restore` CR does not update the `.spec` fields for the CR. 
Backup and restore operations are "one-time" events that modify {brandname} resources.
To update `.spec` fields for the CR, or when a backup or restore operation fails, you must create a new instance of the `Backup` or `Restore` CR.
```

File: documentation/asciidoc/stories/assembly_backing_up_restoring.adoc [hunk 3]
```diff
@@ -8,7 +8,13 @@ ifdef::context[:parent-context: {context}]
 
 `Backup` CR:: Archives {brandname} cluster content to a persistent volume.
 `Restore` CR:: Restores archived content to a {brandname} cluster.
-
++
+[IMPORTANT]
```
Comment (oraNod) [inline documentation/asciidoc/stories/assembly_backing_up_restoring.adoc:12]:
```suggestion
[NOTE]
```

File: documentation/asciidoc/stories/assembly_batch_cr.adoc [hunk 4]
```diff
@@ -6,7 +6,13 @@ ifdef::context[:parent-context: {context}]
 [role="_abstract"]
 {ispn_operator} provides a `Batch` CR that lets you create {brandname} resources in bulk.
 `Batch` CR uses the {brandname} command line interface (CLI) in batch mode to carry out sequences of operations.
-
++
+[IMPORTANT]
```
Comment (oraNod) [inline documentation/asciidoc/stories/assembly_batch_cr.adoc:10]:
```suggestion
[NOTE]
```

File: documentation/asciidoc/stories/assembly_batch_cr.adoc [hunk 5]
```diff
@@ -6,7 +6,13 @@ ifdef::context[:parent-context: {context}]
...
+====
+A `Batch` CR is a one-time event that does not reconcile updates to it's `.spec` fields.
+If the `.spec` needs updating, the execution of the CLI Batch fails or the batch file content changes, then it's necessary for a new `Batch` CR instance to be created.
```
Comment (oraNod) [inline documentation/asciidoc/stories/assembly_batch_cr.adoc:13]:
```suggestion
Applying a `Batch` CR does not update the `.spec` fields for the CR. 
Batch operations are "one-time" events that modify {brandname} resources.
To update `.spec` fields for the CR, or when a batch operation fails, you must create a new instance of the `Batch` CR.
```

**Review by oraNod — APPROVED** [review]
LGTM!

**Comment by oraNod** [comment]
thanks @ryanemerson

**Pull request merged**